applications/nrf_desktop/src/modules/usb_state.c

```diff
@@ -1205,11 +1218,31 @@ static int usb_init(void)
 
 	if (err) {
 		LOG_ERR("Cannot enable USB (err: %d)", err);
+	} else {
+		LOG_DBG("USB enabled");
 	}
 
 	return err;
 }
 
+static void usb_state_module_init(void)
```

Comment:
if we use `usb_init_thread_fn(NULL, NULL, NULL);` in the `app_event_handler` then i think this function can be inlined in the `usb_init_thread_fn`

Reply:
I don't understand. You mean declaring `static inline void usb_init_thread_fn` instead of `static void usb_init_thread_fn`?

Reply:
sorry, I meant to just take content from this function and just put it in the `usb_init_thread_fn` as it is used only there:

```
static void usb_init_thread_fn(void *dummy0, void *dummy1, void *dummy2)
{
	ARG_UNUSED(dummy0);
	ARG_UNUSED(dummy1);
	ARG_UNUSED(dummy2);

	if (usb_init()) {
		module_set_state(MODULE_STATE_ERROR);
	} else {
		module_set_state(MODULE_STATE_READY);
	}
}
```

Reply:
oh, okay. will do.